dotnet/Directory.Build.targets

```diff
@@ -57,9 +68,9 @@
 		<Exec Command="copy $(OutDir)$(ProjectName).deps.json   $(TargetFramework)\$(ProjectName).deps.json" />
 	</Target>
 
-	<Target Name="CopyAssemblies" Condition="'$(ProjectName)'!=''">
-		<Copy SourceFiles="$(ProjectDir)/$(OutDir)$(TargetFileName)" DestinationFolder="$(MSBuildThisFileDirectory)/build/gxnetcore/bin" Condition="'$(TargetFramework)'=='netcoreapp3.1'"/>
-		<Copy SourceFiles="$(ProjectDir)/$(OutDir)$(TargetFileName)" DestinationFolder="$(MSBuildThisFileDirectory)/build/gxnet/bin" Condition="'$(TargetFramework)'!='netcoreapp3.1'"/>
+	<Target Name="CopyAssemblies" Condition="'$(ProjectName)'!='' AND Exists('$(ProjectDir)$(OutDir)')">
+		<Message Importance="high" Text="Copying $(ProjectDir)$(OutDir)$(TargetFileName) to $(DeployDirectory)$(DeployDirectoryPlatformBin)"></Message>
```

Comment:
```suggestion
		<Message Importance="high" Text="Copying $(ProjectDir)\$(OutDir)$(TargetFileName) to $(DeployDirectory)$(DeployDirectoryPlatformBin)"></Message>
```

Reply:
It is not necessary the extra separator. $(ProjectDir) ends with a directory separator. Agree?

Reply:
You have it in the following "copy" target; if it is not neccessary, remove it from "message" and "copy".

Reply:
Removed at commit 344a386